website/server/requestHandlers/LoginHandler.js

```diff
@@ -0,0 +1,35 @@
+const {RequestHandler, RequestResponse } = require("./RequestHandler")
```

Comment:
A space and ';' are missing.
`const { RequestHandler, RequestResponse } = require("./RequestHandler");`